.circleci/config.yml

```diff
@@ -111,7 +111,7 @@ jobs:
             git config user.email "fx-data-platform@mozilla.com"
             git config user.name "CircleCI docs-deploy job"
             npm install -g --silent gh-pages@2.0.1
-            gh-pages -d docs/_build/html
+            gh-pages --dotfiles --message "[skip ci] Updates" --dist docs/_build/html
```

Comment:
The `--dotfiles` flag is necessary for the `.nojekyll` file to get committed. The `[skip ci]` message prevents CircleCI from trying to build the gh-pages branch after this commit (which we expect to fail because it doesn't have the source code).